test/test_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'esp_sdk'
 require 'minitest/autorun'
 require 'minitest/reporters'
+require 'mocha/mini_test'
```

Comment:
We don't have a require mocha anywhere else that I can see. Withy eh active support changes I needed this for tests to work. 



Johnathan Ludwig

On Fri, Feb 13, 2015 at 6:55 AM, Kevin Tyll <notifications@github.com>
wrote:

>> @@ -5,6 +5,7 @@
>>  require 'esp_sdk'
>>  require 'minitest/autorun'
>>  require 'minitest/reporters'
>> +require 'mocha/mini_test'
> Why mocha/minitest?
> Do we still need require/mocha?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/EvidentSecurity/esp_sdk/pull/16/files#r24659680